packages/@react-aria/interactions/src/usePress.ts

```diff
@@ -262,278 +272,295 @@ export function usePress(props: PressHookProps): PressResult {
       }
     };
 
-    if (typeof PointerEvent !== 'undefined') {
-      pressProps.onPointerDown = (e) => {
-        // Only handle left clicks
-        if (e.button !== 0) {
-          return;
-        }
+    // if (typeof PointerEvent !== 'undefined') {
```

Comment:
Can we remove the commented out code because it exists in the repo?

Reply:
Potentially, it does serve as a nice reminder, of what we're aiming for. But I don't have a strong opinion one way or the other

Reply:
Talked to team, we're leaving them in as comment for now as a clear message to anyone looking at the code for why we aren't using pointerevents. Also letting it serve as a reminder to bring the code back.